src/core/zowe/core_for_zowe_sdk/profile_manager.py

```diff
@@ -253,8 +333,34 @@ def load(
 
         If `profile_type` is not base, then we will load properties from both
         `profile_type` and base profiles and merge them together.
-        """
 
+        Parameters
+        ----------
+        profile_name : Optional[str]
+            The name of the profile to load. If None, profiles are loaded based only on profile type.
+        profile_type : Optional[str]
+            The type of the profile to load, e.g., 'user', 'admin'. If None, profiles are loaded based only on name.
```

Comment:
Do we want to provide valid examples for the profile types to load?

```suggestion
            The type of the profile to load, e.g., 'zosmf', 'zftp'. If None, profiles are loaded based only on name.
```